docs/quickstarts/aws.md

```diff
@@ -201,11 +253,11 @@ module.aws-myawscluster.null_resource.copy-controller-secrets: Still creating...
 The error probably happens because the `ssh_pubkeys` provided in the configuration is missing in the
 `ssh-agent`.
 
-To rectify the error, you need to:
+In case the deployment process seems to hang at the `copy-controller-secrets` phase for a long
+time, check the following:
 
-1. Follow the steps [to add the SSH key to the
-   ssh-agent](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent).
-2. Retry [Step 5](#step-5-create-lokomotive-cluster).
+- Verify the correct private SSH key was added to `ssh-agent`.
+- Verify that you can SSH into the created controller node from the machine running `lokoctl`.
 
 ### IAM Permission Issues
```

Comment:
here we should point to the IAM permissiosn specified in DNS document: https://github.com/kinvolk/lokomotive/blob/aee8f1e13021fda65a10e8bcdc8e246bd528b618/docs/concepts/dns.md#aws-route-53 rather than pointing to the generic AWS documentation.